Mathlib/FieldTheory/Relrank.lean

```diff
@@ -35,11 +35,13 @@ variable {E : Type v} [Field E] {L : Type w} [Field L]
 
 variable (A B C : Subfield E)
 
+set_option synthInstance.maxHeartbeats 400000 in
```

Comment:
Can we add some adaptation notes here? Or is this just a fact of life that we have to live with until we debug/speedup?